releasenotes/notes/quandl-discontinued-92f3a1e020e6ff9c.yaml

```diff
@@ -0,0 +1,5 @@
+---
+other:
+  - |
+    Quandl python client has been discontinued and was replaced by Nasdaq Data Link.
+    More information at https://pypi.org/project/Nasdaq-Data-Link/
```

Comment:
```suggestion
    Data providers using of `Quandl` have been switched to `Nasdaq Data Link` since the
    Quandl python client has been discontinued and was replaced by Nasdaq Data Link.
    More information at https://pypi.org/project/Nasdaq-Data-Link/
```
Can we say something like this so its clear for a user of Qiskit Finance what the impact is. I am not sure if we need any upgrade note to suggest that is they had and were using a Quandl token that needs to be altered to a Nasdaq Data Link token now for these updated data providers?

Reply:
The token is the same. I didn't change mine. In fact the previous quandl url redirects you to the new url and your userid/password and token are preserved.